packages/lit-dev-content/site/tutorials/content/svg-templates/05.md

```diff
@@ -0,0 +1,119 @@
+### Overview
+
+This is chapter of the demo covers the basics of CSS in SVG including
+how to:
+
+- use CSS to style SVG
+- use CSS Custom Properties to create a theme
+
+### Learn
+
+CSS can apply attributes to SVG elements in the same manner CSS
+applies attributes to HTML.
+
+However, unlike HTML, only [SVG Presentation Attributes](https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/SVG_and_CSS)
```

Comment:
That link does not define "SVG Presentation Attributes". Perhaps define that term and then link out to a "read more on svg and css here"

Reply:
done!